NEWS.md

```diff
@@ -0,0 +1,7 @@
+# fhpredict 0.0.0.9000
+
+* Added a `NEWS.md` file to track changes to the package.
```

Comment:
IMHO should be `changelog.md` not NEWS.md

Reply:
Can't say anything about tidyverse. If it is like this in R package land pls ignore

Reply:
This file is required (or at least recognised) by the R packaging mechanism and will show up in the help browser.